pkg/simple/client/openpitrix/helmwrapper/helm_wrapper.go

```diff
@@ -258,60 +192,45 @@ func SetMock(mock bool) Option {
 	}
 }
 
+func NewInClusterRESTClientGetter(kubeconfig string, namespace string) genericclioptions.RESTClientGetter {
+	flags := genericclioptions.NewConfigFlags(true)
+	if kubeconfig != "" {
+		cfg, _ := clientcmd.RESTConfigFromKubeConfig([]byte(kubeconfig))
+		flags = genericclioptions.NewConfigFlags(false)
+		flags.APIServer = &cfg.Host
+		flags.BearerToken = &cfg.BearerToken
+		flags.CAFile = &cfg.CAFile
+		flags.WithDiscoveryBurst(cfg.Burst)
+		if sa := cfg.Impersonate.UserName; sa != "" {
+			flags.Impersonate = &sa
+		}
+	}
```

Comment:
The following code will cause a cert error if the kubefonfig includes certificate-authority-data.

```
E0513 11:24:17.485511       1 helm_wrapper.go:107] namespace: z1, name: redis-emjr1v, run command failed, error: Kubernetes cluster unreachable: Get "https://10.233.0.1:443/version?timeout=32s": x509: certificate signed by unknown authority
```

Reply:
Wrapped  RESTClientGetter
```
type restClientGetter struct {
	configBytes []byte
}

func (r *restClientGetter) ToRawKubeConfigLoader() clientcmd.ClientConfig {
	config, err := clientcmd.NewClientConfigFromBytes(r.configBytes)
	if err != nil {
		klog.Errorf("failed to create client from bytes: %s", r.configBytes)
	}
	return config
}

func (r *restClientGetter) ToRESTConfig() (*rest.Config, error) {
	return r.ToRawKubeConfigLoader().ClientConfig()
}

func (r *restClientGetter) ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error) {
	config, err := r.ToRESTConfig()
	if err != nil {
		return nil, err
	}
	discoveryClient, err := discovery.NewDiscoveryClientForConfig(config)
	if err != nil {
		return nil, err
	}
	clientset := memory.NewMemCacheClient(discoveryClient)
	return clientset, nil
}
```

Reply:
Wrapped  RESTClientGetter
```
type restClientGetter struct {
	configBytes []byte
}

func (r *restClientGetter) ToRawKubeConfigLoader() clientcmd.ClientConfig {
	config, err := clientcmd.NewClientConfigFromBytes(r.configBytes)
	if err != nil {
		klog.Errorf("failed to create client from bytes: %s", r.configBytes)
	}
	return config
}

func (r *restClientGetter) ToRESTConfig() (*rest.Config, error) {
	return r.ToRawKubeConfigLoader().ClientConfig()
}

func (r *restClientGetter) ToDiscoveryClient() (discovery.CachedDiscoveryInterface, error) {
	config, err := r.ToRESTConfig()
	if err != nil {
		return nil, err
	}
	discoveryClient, err := discovery.NewDiscoveryClientForConfig(config)
	if err != nil {
		return nil, err
	}
	clientset := memory.NewMemCacheClient(discoveryClient)
	return clientset, nil
}
```